ansible/roles/csm.rr.ceph_zoning/files/ceph_zoning.py

```diff
@@ -127,7 +104,7 @@ def create_and_apply_rules() -> None:
         logger.debug(f"Applying new rule to pool: {pool}")
         run_command(f"ceph osd pool set {pool} crush_rule replicated_rule_with_rack_failure_domain")
 
-def service_zoning(positions_dict: Dict[str, List[str]], sn_count_in_rack: int) -> None:
+def service_zoning(positions_dict: Dict[str, List[str]], sn_count_in_rack: list) -> None:
```

Comment:
Should also update the description of `sn_count_in_rack` in the function docstring
```suggestion
def service_zoning(positions_dict: Dict[str, List[str]], sn_count_in_rack: List[int[) -> None:
```

Reply:
Did you mean `List[int])`?
```suggestion
def service_zoning(positions_dict: Dict[str, List[str]], sn_count_in_rack: List[int]) -> None:
```